### PR TITLE
Org page to react

### PR DESCRIPTION
**Pull request by peterholmberg**
Converted org settings page to react.

- Created a new component, `<SimplePicker />` which can be used for basic drop downs with static values. With this we can have the same look on all dropdowns.

- There's at the moment two tooltip components in Grafana. `<ToolTip />` and `<Popover />` where the latter is not used anywhere. Both created by @jschill . Maybe you can add some insight?

### CODE REVIEW

File: public/app/features/org/OrgPreferences.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import {
+  setOrganizationHomeDashboard,
+  setOrganizationTheme,
+  setOrganizationTimezone,
+  updateOrganizationPreferences,
+} from './state/actions';
+
+export interface Props {
+  preferences: OrganizationPreferences;
+  starredDashboards: DashboardAcl[];
```
Comment (torkelo) [inline public/app/features/org/OrgPreferences.tsx]: this has be the wrong type,

File: public/app/features/org/OrgPreferences.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            defaultValue={themes.find(theme => theme.value === preferences.theme)}
+            options={themes}
+            getOptionValue={i => i.value}
+            getOptionLabel={i => i.text}
+            onSelected={theme => setOrganizationTheme(theme.value)}
+            width={20}
+          />
+        </div>
+        <div className="gf-form">
+          <span className="gf-form-label width-11">
```
Comment (torkelo) [inline public/app/features/org/OrgPreferences.tsx]: Can we use the label component instead? https://github.com/grafana/grafana/blob/master/public/app/core/components/Label/Label.tsx

**Review by torkelo — CHANGES_REQUESTED** [review]
Great stuff!  the biggest thing that stands out is the use of DashboardAcl type (dashboard permission list item) for starred dashboards type. This should be SearchHit type think or DashboardSearchHit

File: public/app/features/org/OrgPreferences.tsx (hunk 1 above)
Comment (peterholmberg) [reply public/app/features/org/OrgPreferences.tsx]: Yeah, forgot to change it. Used Acl as placeholder. Should we go with DashboardSearchHit?

File: public/app/types/dashboard.ts [hunk 3]
```diff
@@ -1,5 +1,15 @@
 import { DashboardAcl } from './acl';
 
+export interface Dashboard {
```
Comment (torkelo) [inline public/app/types/dashboard.ts]: This type is specific to the search api, can you rename this to DashboardSearchHit and move it to search.ts?

File: public/app/types/organization.ts [hunk 4]
```diff
@@ -14,5 +14,5 @@ export interface OrganizationPreferences {
 export interface OrganizationState {
   organization: Organization;
   preferences: OrganizationPreferences;
-  starredDashboards: DashboardAcl[];
+  starredDashboards: Dashboard[];
```
Comment (torkelo) [inline public/app/types/organization.ts]: Not sure this should be part of the organization state as the starred dashboard is part of the user (starring is user action) so maybe it is better to be part of a user state (would like to move some of the contextSrv.user stuff to a user redux store)

**Comment by jschill** [comment]
@peterholmberg Unfortunately no insights in usage. Looking at the initial issue it seems like we just created the component, without any use-case? #10389 PR #10473

(The thing that differs in the component is that one reacts on clicks and one on mouseover)

**Pull request merged**